frames/src/main/java/com/checkout/frames/component/expirydate/ExpiryDateComponent.kt

```diff
@@ -30,10 +47,43 @@ internal fun ExpiryDateComponent(
 @Preview(showBackground = true)
 @Composable
 private fun ExpiryDateComponentPreview() {
-    ExpiryDateComponent(
-        ExpiryDateComponentStyle(
-            InputComponentStyle(inputFieldStyle = InputFieldStyle(placeholderText = "Test placeholder"))
+    val style = InputComponentStyle(
+        titleStyle = TextLabelStyle(
+            "Expiry Date",
+            textStyle = TextStyle(size = 16, color = 0xFF141414, font = Font.SansSerif, maxLines = 1),
```

Comment:
Question : Why are we hardcoding size, color,  font here. Are they default values and can be changed by merchant ?

Reply:
Yes, it is created as a preview for us to test this component. Yes, it can be changed.